src/engines/LeastFrequentlyUsed.js

```diff
@@ -0,0 +1,123 @@
+import DefaultHashTable from '../dataStructure/HashTable';
+import DoublyLinkedList from '../dataStructure/DoublyLinkedList';
+import {
+  add as hAdd,
+  get as hGet,
+  has as hHas,
+  remove as hRemove,
+} from '../hashTableSymbol';
+
+function LeastFrequentlyUsed({
+  size = Number.MAX_SAFE_INTEGER,
+  HashTable = DefaultHashTable,
+}) {
+  if (size <= 0) throw Error('LRU size specified should be greater than zero');
```

Comment:
null makes it true and is not replaced by default value ;-)
Sugestion validator function 'isPositiveInteger'